client/components/App.jsx

```diff
@@ -39,6 +40,23 @@ class App extends Component {
     zoomToken: '',
   };
 
+  static getInterviewDetails = (description) => {
+    // Determine whether the interview is based on the newer format
+    const isNewer = description.includes(interviewsNewMetadata) >= 0;
```

Comment:
`includes` returns a Boolean, so it replaces the `indexOf(foo) >= 0` workaround entirely. This partial replacement doesn't work, since true == 1 and false == 0.